float64 rollAng
float64 pitchAng1
float64 pitchAng2